cibuildwheel/options.py

```diff
@@ -290,7 +298,8 @@ def get(
         """
 
         if name not in self.default_options and name not in self.default_platform_options:
-            raise ConfigOptionError(f"{name} must be in cibuildwheel/resources/defaults.toml file")
+            msg = f"{name} must be in cibuildwheel/resources/defaults.toml file to be accessed."
```

Comment:
Any specific reason not to use the following in this instance ?
```suggestion
            msg = f"{name!r} must be in cibuildwheel/resources/defaults.toml file to be accessed."
```

I guess it would also be nice to get it with quotes ?

Reply:
No, it just didn't have quotes before.